File: 4_August_Batch/Java/Factorial.java
import java.util.*;
public class Factorial
{
    public static void main(String args[])
    {   
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter a number: ");
        int a = sc.nextInt();
        int fact=1;
        while(a!=0){
            fact=fact*a;
            a--;
        }
        System.out.println(fact);
        
    }
}

File: 11_November_Batch/C++/factorial.cpp
#include<iostream>
using namespace std;
int factorial(int a){
    int fact=1;
    while(a!=0){
        fact*=a;
        a--;
    }
    return fact;
}
int main(){
    int a , ans;
    cin>>a;
    ans = factorial(a);
    cout<<ans;
    return 0;
}